src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -248,9 +248,18 @@ public IFMLSidedHandler getSidedDelegate()
         return sidedDelegate;
     }
 
+    /**
+     * @deprecated use {@link #onPostServerTick(MinecraftServer)}
+     */
+    @Deprecated
     public void onPostServerTick()
     {
-        bus().post(new TickEvent.ServerTickEvent(Phase.END));
+        this.onPreServerTick(this.getMinecraftServerInstance());
```

Comment:
I've fixed this locally, will push in a few minutes once `genPatches` finishes - thanks